### PR TITLE
[DE] Add `History of osu!/Online rankings`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/History_of_osu!/Online_rankings/de.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Frühe Versionen von osu! waren meistens ein einsames Vergnügen ohne Online-Bestenlisten, bis am [6. Oktober 2007](https://osu.ppy.sh/community/forums/topics/15?n=29) öffentliche Online-Ranglisten für [osu!](/wiki/Game_mode/osu!) hinzugefügt wurden. Seitdem haben alle [Spielmodi](/wiki/Game_mode) eigene [Bestenlisten](https://osu.ppy.sh/rankings/osu/performance) erhalten.
+
+Zahlreiche Spieler hielten den ersten Rang in jedem Spielmodus mit verschiedenen Ranglistensystemen. Das aktuelle System, das von allen Spielmodi für Online-Ranglisten verwendet wird, heißt [ppv2](/wiki/Performance_points/ppv2).
+
+## ![][osu!] osu!
+
+*Hauptseite: [osu!](osu!)*
+
+Seit dem 19. Juli 2021 ist ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902/osu) auf Platz 1 der [osu! Performance-Rangliste](https://osu.ppy.sh/rankings/osu/performance).
```
Comment (respektive) [inline wiki/History_of_osu!/Online_rankings/de.md:24]: He's been #1 for longer, this is just the date this line got updated i guess.
```suggestion
Mit Stand vom 19. Juli 2021 ist ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902/osu) auf Platz 1 der [osu! Performance-Rangliste](https://osu.ppy.sh/rankings/osu/performance).
```

File: wiki/History_of_osu!/Online_rankings/de.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## ![][osu!taiko] osu!taiko
+
+*Hauptseite: [osu!taiko](osu!taiko)*
+
+Seit dem 4. Oktober 2022 ist ::{ flag=JP }:: [shinchikuhome](https://osu.ppy.sh/users/3174184/taiko) auf Platz 1 der [osu!taiko Performance-Rangliste](https://osu.ppy.sh/rankings/taiko/performance).
```
Comment (respektive) [inline wiki/History_of_osu!/Online_rankings/de.md:30]:
```suggestion
Mit Stand vom 4. Oktober 2022 ist ::{ flag=JP }:: [shinchikuhome](https://osu.ppy.sh/users/3174184/taiko) auf Platz 1 der [osu!taiko Performance-Rangliste](https://osu.ppy.sh/rankings/taiko/performance).
```

File: wiki/History_of_osu!/Online_rankings/de.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## ![][osu!catch] osu!catch
+
+*Hauptseite: [osu!catch](osu!catch)*
+
+Seit dem 7. Juni 2021 ist ::{ flag=KR }:: [YesMyDarknesss](https://osu.ppy.sh/users/4158549/fruits) auf Platz 1 der [osu!catch Performance-Rangliste](https://osu.ppy.sh/rankings/fruits/performance).
```
Comment (respektive) [inline wiki/History_of_osu!/Online_rankings/de.md:36]:
```suggestion
Mit Stand vom 7. Juni 2021 ist ::{ flag=KR }:: [YesMyDarknesss](https://osu.ppy.sh/users/4158549/fruits) auf Platz 1 der [osu!catch Performance-Rangliste](https://osu.ppy.sh/rankings/fruits/performance).
```

File: wiki/History_of_osu!/Online_rankings/de.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## ![][osu!mania] osu!mania
+
+*Hauptseite: [osu!mania](osu!mania)*
+
+Seit dem 13. April 2023 ist ::{ flag=KR }:: [dressurf](https://osu.ppy.sh/users/758406/mania) auf Platz 1 der [osu!mania Performance-Rangliste](https://osu.ppy.sh/rankings/mania/performance).
```
Comment (respektive) [inline wiki/History_of_osu!/Online_rankings/de.md:42]:
```suggestion
Mit Stand vom 13. April 2023 ist ::{ flag=KR }:: [dressurf](https://osu.ppy.sh/users/758406/mania) auf Platz 1 der [osu!mania Performance-Rangliste](https://osu.ppy.sh/rankings/mania/performance).
```

**Review by respektive — CHANGES_REQUESTED** [review]

**Comment by The-Last-Cookie** [comment]
@respektive I dug a bit through the internet and you're right. mrekk got the 1st place on 8th April, 2021:

- [Reddit thread 1](https://www.reddit.com/r/osugame/comments/mmkaag/mrekk_is_now_1_surpassing_whitecat/)
- [Reddit thread 2](https://www.reddit.com/r/osugame/comments/mmkf3p/mrekk_becoming_1_means_a_lot_of_things_the/)
- [tweet from mrekk](https://twitter.com/mrekkosu/status/1380024575136722944)

**Review by respektive — APPROVED** [review]

**Pull request merged**